main.nf

```diff
@@ -553,250 +531,266 @@ if (compressedReference) {
-        file additional_fasta from ch_additional_fasta_to_concat.collect()
-        file additional_gtf from ch_additional_gtf.collect()
+        path genome_fasta from ch_genome_fasta
+        path genome_gtf from gtfFile
+        path additional_fasta from ch_additional_fasta_to_concat.collect()
+        path additional_gtf from ch_additional_gtf.collect()
 
         output:
-        file "${genome_name}.fa" into (
-            ch_fasta_for_star_index, ch_fasta_for_hisat_index, 
-            ch_fasta_for_salmon_transcripts, ch_fasta_for_rsem_reference
-        )
-        file "${genome_name}.gtf" into (
-            gtf_makeSTARindex, gtf_makeHisatSplicesites, gtf_makeHISATindex,
-            gtf_makeSalmonIndex, gtf_makeBED12, gtf_star, gtf_dupradar, gtf_featureCounts,
-            gtf_stringtieFPKM, gtf_salmon, gtf_salmon_merge, gtf_qualimap,
-            gtf_makeRSEMReference
-        )
+        path "${genome_name}.fa" into ch_fasta_for_star_index,
+                                      ch_fasta_for_hisat_index,
+                                      ch_fasta_for_salmon_transcripts,
+                                      ch_fasta_for_rsem_reference
+        path "${genome_name}.gtf" into gtf_makeSTARindex,
+                                       gtf_makeHisatSplicesites,
+                                       gtf_makeHISATindex,
+                                       gtf_makeSalmonIndex,
+                                       gtf_makeBED12,
+                                       gtf_star,
+                                       gtf_dupradar,
+                                       gtf_featureCounts,
+                                       gtf_stringtieFPKM,
+                                       gtf_salmon,
+                                       gtf_salmon_merge,
+                                       gtf_qualimap,
+                                       gtf_makeRSEMReference
 
         script:
         main_genome_name = params.genome ? params.genome : genome_fasta.getBaseName()
-
         transgenomes = additional_fasta.collect{ it.getBaseName() }.sort().join("+")
-        genome_name = "${main_genome_name}__${transgenomes}"
+        genome_name = "${main_genome_name}_${transgenomes}"
         """
         cat $genome_fasta $additional_fasta > ${genome_name}.fa
         cat $genome_gtf $additional_gtf > ${genome_name}.gtf
         """
     }
 } else {
-    gtfFile.into{ gtf_makeSTARindex; gtf_makeHisatSplicesites; gtf_makeHISATindex;
-        gtf_makeSalmonIndex; gtf_makeBED12; gtf_star; gtf_dupradar; gtf_featureCounts;
-        gtf_stringtieFPKM; gtf_salmon; gtf_salmon_merge; gtf_qualimap; 
-        gtf_makeRSEMReference }
+    gtfFile
+        .into { gtf_makeSTARindex; gtf_makeHisatSplicesites; gtf_makeHISATindex;
+                gtf_makeSalmonIndex; gtf_makeBED12; gtf_star; gtf_dupradar; gtf_featureCounts;
+                gtf_stringtieFPKM; gtf_salmon; gtf_salmon_merge; gtf_qualimap; gtf_makeRSEMReference }
```

Comment:
```suggestion
        .into { gtf_makeSTARindex
                 gtf_makeHisatSplicesites
                 gtf_makeHISATindex
                 gtf_makeSalmonIndex
                 gtf_makeBED12
                 gtf_star
                 gtf_dupradar
                 gtf_featureCounts
                 gtf_stringtieFPKM
                 gtf_salmon
                 gtf_salmon_merge
                 gtf_qualimap
                 gtf_makeRSEMReference }
```

Reply:
just to follow the same structure then

Reply:
Sorted https://github.com/nf-core/rnaseq/pull/455/commits/67096678b45d4df032b69cb116e37e4af7a5a917